package.json

```
@@ -22,13 +22,13 @@
     "build:bp:composables": "cd packages/boilerplate/composables && yarn build",
     "build:bp:helpers": "cd packages/boilerplate/helpers && yarn build",
     "build:bp:theme": "cd packages/boilerplate/theme && yarn build",
-    "test:ct:api-client": "cd packages/commercetools/api-client && yarn test",
-    "test:ct:composables": "cd packages/commercetools/composables && yarn test",
-    "test:ct:helpers": "cd packages/commercetools/helpers && yarn test",
-    "test:core": "jest -c ./packages/core/jest.config.js",
-    "test:prismic": "cd packages/prismic && yarn test",
-    "test:ct": "yarn test:ct:api-client && yarn test:ct:composables && yarn test:ct:helpers",
-    "test:ct:ci": "yarn test:ct:api-client && yarn test:ct:composables && yarn test:ct:helpers",
+    "test:ct:api-client": "cd packages/commercetools/api-client && yarn test --coverage",
+    "test:ct:composables": "cd packages/commercetools/composables && yarn test --coverage",
+    "test:ct:helpers": "cd packages/commercetools/helpers && yarn test --coverage",
+    "test:core": "jest -c ./packages/core/jest.config.js --coverage",
+    "test:prismic": "cd packages/prismic && yarn test --coverage",
+    "test:ct": "yarn test:ct:api-client && yarn test:ct:composables && yarn test:ct:helpers --coverage",
+    "test:ct:ci": "yarn test:ct:api-client && yarn test:ct:composables && yarn test:ct:helpers --coverage",
```

```suggestion
    "test:ct": "yarn test:ct:api-client && yarn test:ct:composables && yarn test:ct:helpers",
    "test:ct:ci": "yarn test:ct:api-client && yarn test:ct:composables && yarn test:ct:helpers",
```

`--coverage` in these two lines is redundant :)

---

True, good eye!